### PR TITLE
workflow with more than one dir and not covered functions

### PR DESCRIPTION
**Pull request by jorgepz**
Hi, I am trying to understand the testing and codecoverage functionality for cases with more than one directory and test functions. I think these changes in the workflows might be helpful for new users. I also added a not covered function to test the codecoverage percentage, however I am not being able to get the codecoverage to work, I will keep trying but any help or suggestions would be well received!

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -127,7 +127,7 @@ jobs:
     - name: Run unit tests
       uses: ./
       with:
-        src: tests/src
+        src: tests/src tests/srcB
```
Comment (joergbrech) [inline .github/workflows/test.yml:130]: I would prefer if you added a new test that tests your behavior, instead of changing every test.

**Review by joergbrech — CHANGES_REQUESTED** [review]
Thanks a lot for your PR! Could you explain exactly the discrepancy between the expected behavior and the actual behavior? That is, could you please be more specific with regard to what you mean by 
>I am not being able to get the codecoverage to work

Thanks

**Comment by joergbrech** [comment]
Note, that by design the code coverage reported here does not respect your changes, see

**Comment by joergbrech** [comment]
Sorry, my 1 1/2 year old daughter just grabbed the mouse and closed this PR, this was not intentional :laughing:

**Comment by jorgepz** [comment]
> Thanks a lot for your PR! Could you explain exactly the discrepancy between the expected behavior and the actual behavior? That is, could you please be more specific with regard to what you mean by
> 
> > I am not being able to get the codecoverage to work
> 
> Thanks

Hi @joergbrech thanks for your comments and your time (even while taking care of your daughter 🤣  ) .

My goal is to add coverage to [this repo](https://github.com/ONSAS/ONSAS.m). I am already using moxunit-action for testing and I think codecoverage would be a great thing to add!

My repo tests use more than one src directory https://github.com/ONSAS/ONSAS.m/blob/master/.github/workflows/tests.yml , then... I was trying to add more than one directory to moxunit-action test.yml to figure out how should I edit mine... and by the way try to help other newbie Octave developers like me 😃  

Best,
Jorge

File: .github/workflows/test.yml (hunk 1 above)
Comment (jorgepz) [reply .github/workflows/test.yml]: ok. this was back to normal in 90d5363

**Comment by jorgepz** [comment]
Hi. I've been working in a small test repo and got this minimal working example that might help to show what I think would be nice to add to some test. In this PR https://github.com/jorgepz/Octave_Matlab_CI_Testbed/pull/1 I add a call to a function in another directory, then, in the .yml file I had to add the new folder srcB to the src line...
```
        src: tests/src tests/srcB
```

**Comment by jorgepz** [comment]
I am reading that there is a limitation for MOcov using more than one directory https://github.com/MOcov/MOcov . I'll close this draftPR, take a look at the MOcov repo and try to fix my yml after that.
Thanks for your time @joergbrech !

**Pull request closed without merging**